packages/material-ui/src/CardContent/CardContent.js

```diff
@@ -37,7 +37,7 @@ CardContent.propTypes = {
    * The component used for the root node.
    * Either a string to use a HTML element or a component.
    */
-  component: PropTypes.elementType,
+  component: PropTypes /* @typescript-to-proptypes-ignore */.elementType,
```

Comment:
Proptypes arent't generated yet. It's just in preparation of future work and was easy to find&replace.